src/applications/travel-pay/components/TravelClaimDetails.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useState } from 'react';
+import { useParams, useLocation } from 'react-router-dom';
+import { scrollToTop } from 'platform/utilities/ui/scroll';
+import { focusElement } from 'platform/utilities/ui';
+
+import { apiRequest } from '@department-of-veterans-affairs/platform-utilities/api';
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
+
+import { formatDateTime } from '../util/dates';
+import BreadCrumbs from './Breadcrumbs';
+
+export default function TravelClaimDetails() {
+  const { id } = useParams();
+  const location = useLocation();
+  const { claimDetailsProps = null } = location.state ?? {};
+  const [claimDetails, setClaimDetails] = useState(claimDetailsProps);
+  const [isLoading, setIsLoading] = useState(!claimDetailsProps);
+  const [claimsError, setClaimsError] = useState(null);
+  const BTSSS_PORTAL_URL = 'https://dvagov-btsss.dynamics365portals.us/';
+
+  console.log(claimDetails); // eslint-disable-line no-console
+  useEffect(() => {
+    // window.scrollTo(0, 0);
+    // scrollTo('h1');
+    focusElement('h1');
+    scrollToTop();
+  });
+  useEffect(
+    () => {
+      const fetchClaim = async () => {
+        try {
+          const claimsUrl = `${environment.API_URL}/travel_pay/v0/claims/${id}`;
+          const claimsResponse = await apiRequest(claimsUrl);
+          setClaimDetails(claimsResponse);
+          setIsLoading(false);
+        } catch (error) {
+          setClaimsError(error);
+        }
+      };
+
+      if (!claimDetails) {
+        fetchClaim();
+      }
+    },
+    [claimDetails, id],
+  );
+
+  if (isLoading) {
+    return (
+      <div className="vads-l-grid-container vads-u-padding-y--3">
+        <va-loading-indicator
+          label="Loading"
+          message="Please wait while we load the application for you."
+          data-testid="travel-pay-loading-indicator"
+        />
+      </div>
+    );
+  }
+
+  const {
+    createdOn,
+    claimStatus,
+    claimNumber,
+    appointmentDateTime,
+    facilityName,
+    modifiedOn,
+  } = claimDetails || {};
+
+  const [appointmentDate, appointmentTime] = formatDateTime(
+    appointmentDateTime,
+  );
+  const [createDate, createTime] = formatDateTime(createdOn);
+  const [updateDate, updateTime] = formatDateTime(modifiedOn);
+
+  return (
+    <>
+      {/* Q: Is the Mhv nav needed here? */}
+      {/* <MhvSecondaryNav /> */}
+
+      <article className="usa-grid-full vads-u-padding-bottom--0">
+        {/* <div className="claim-details-breadcrumb-wrapper"> */}
+        {/* <va-icon className="back-arrow" icon="arrow_back" /> */}
+        <BreadCrumbs />
+        {claimDetails && (
+          <>
+            <h1>Your travel reimbursement claim for {appointmentDate}</h1>
+            <h2>Claim number: {claimNumber}</h2>
```

Comment:
This variable name is correct, but the screenshot shows a UUID-formatted claim ID. This is not a useful ID to the Veteran as the UUID is an internal-to-btsss id. The claim number should have the format `TC[0-9]{13}`, eg `TC0123456789123`. I suspect a mock data issue, but something to look into.